src/ostree/ot-builtin-commit.c

```diff
@@ -94,7 +94,7 @@ parse_fsync_cb (const char  *option_name,
  */
 
 static GOptionEntry options[] = {
-  { "parent", 0, 0, G_OPTION_ARG_STRING, &opt_parent, "Parent ref, or \"none\"", "REF" },
+  { "parent", 0, 0, G_OPTION_ARG_STRING, &opt_parent, "Parent checksum, or \"none\"", "CHECKSUM" },
```

Comment:
Thinking a bit more about it, can we use `COMMIT` here? Then we can use the `CHECKSUM` literal for arbitrary object types, e.g. as processed by `ostree show`...

Reply:
Yep agree

Reply:
Fixed!